src/main/groovy/com/ullink/ProjectFileParser.groovy

```diff
@@ -75,6 +75,16 @@ class ProjectFileParser {
         }
     }
 
+    File findReferencedDll(String  dllName) {
+        def dllPath = references.find {
+            it.Filename.startsWith(dllName)

```

Comment:
`find`takes the first occurrence matching its condition. If you need something more specific, can't you just input more characters matching the DLL?